### PR TITLE
Update 3 publications and some name fixes

### PR DESCRIPTION
**Pull request by mrshirts**
Updated publication information for 3 publications, plus adding middle initials to match authors in publication.

### CODE REVIEW

**Comment by mrshirts** [comment]
Wasn't sure to check how things appeared in actual website form, just updated the YAML

File: data/publications/index.yaml [hunk 1]
```diff
@@ -19,6 +19,13 @@
...
     license: "CC BY 4.0"
     date: 2022-11-04
+   published:
+    doi: https://doi.org/10.1021/acs.jcim.2c01185
+    journal: Journal of Chemical Information and Modeling
+    volume: 62:23
+    pages: 6094–6104
+    year: 2022
+    date: 2022-11-26
   code: openforcefield/qca-dataset-submission/tree/master/submissions/2021-07-28-OpenFF-Industry-Benchmark-Season-1-MM-v1.1
```
Comment (Yoshanuikabundi) [inline data/publications/index.yaml]:
```suggestion
  published:
    doi: https://doi.org/10.1021/acs.jcim.2c01185
    journal: Journal of Chemical Information and Modeling
    volume: 62:23
    pages: 6094–6104
    year: 2022
    date: 2022-11-26
  code: openforcefield/qca-dataset-submission/tree/master/submissions/2021-07-28-OpenFF-Industry-Benchmark-Season-1-MM-v1.1
```

This line is incorrectly indented :)

**Review by Yoshanuikabundi — COMMENTED** [review]
Hi @mrshirts - you can check the rendered website locally by installing Hugo and running `hugo server -D`. This PR is great, thanks, there was just one indentation error that caused it to fail to build.

We also need to fix our CI! I think the site is served with GitHub Pages but the PR CI hasn't been updated from Travis. I'll open an issue.

**Comment by Yoshanuikabundi** [comment]
Also - I *thoroughly* enjoyed your recent LiveCoMS review on enhanced sampling methods!

**Comment by mrshirts** [comment]
> Also - I thoroughly enjoyed your recent LiveCoMS review on enhanced sampling methods!

Excellent!  Please leave issues for anything you'd like to see differently!

**Comment by jchodera** [comment]
@mrshirts : The espaloma paper has been published too. Can you add:
```YAML
  published:
    doi: https://doi.org/10.1039/D2SC02739A
    journal: Chemical Science
    volume: 13
    pages: 12016-12033
    year: 2022
    date: 2022-09-08
```

**Comment by Yoshanuikabundi** [comment]
Ok I've gone ahead and fixed the indentation issue and updated the espaloma paper (including updating the title and author list). Thanks for sending this in @mrshirts  @jchodera!

**Comment by mrshirts** [comment]
Thanks, @Yoshanuikabundi !

**Pull request merged**